ddpui/core/dashboard_chat/runtime.py

```
@@ -0,0 +1,960 @@
+            state["warnings"] = state.get("warnings", []) + [
+                "No dashboard tables were available for schema inspection.",
+            ]
+            return state
+
+        try:
+            warehouse_tools = self.warehouse_tools_factory(state["org"])
+            schema_snippets = warehouse_tools.get_schema_snippets(candidate_tables)
+        except DashboardChatWarehouseToolsError as error:
+            state["schema_snippets"] = {}
+            state["schema_prompt"] = ""
+            state["warnings"] = state.get("warnings", []) + [str(error)]
```

Ruff (0.15.6)

[warning] Consider `[*state.get("warnings", []), str(error)]` instead of concatenation

Replace with `[*state.get("warnings", []), str(error)]`

(RUF005)